### ISSUE TITLE
NameError (uninitialized constant Cell::ViewModel)

### ISSUE THREAD

**Issue by mensfeld** [trailblazer/trailblazer-rails#38]
When using trbr and trbr-rails without cells included it fails on that line https://github.com/trailblazer/trailblazer-rails/blob/master/lib/trailblazer/rails/controller.rb#L37 as the constant is not defined

**Comment by v-kolesnikov** [comment]
As workaround, just add to your Gemfile: `gem 'trailblazer-cells'`.

**Comment by mensfeld** [comment]
Yeah - I just created an empty ::Cell::ViewModel class to just avoid adding a next gem ;)
thank you!

**Comment by mensfeld** [comment]
Thank you. Great job!
